### PR TITLE
Feature/homepage comp

### PR DESCRIPTION
**Pull request by Gabby-Recny**
### What (if any) features are you implementing?
- Styling for Cards and fix header layout
### What (if anything) did you refactor?
- N/A
### Were there any issues that arose?
- N/A
### Is there anything that you need from your teammate?
- Code review and merge
### Refactor checklist
- [ X] Check variable and parameter names: are they descriptive and unique?
- [X ] Is the code DRY? SRP?
- [ X] Is it easy for others to read your code? (explanatory comments needed?)
- [ X] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [X ] Are commit messages formatted correctly?

### CODE REVIEW

File: src/Cards.css [hunk 1]
```diff
@@ -1,5 +1,23 @@
 .movie-cards {
-    width: 20px;
-    height: 20px;
-    padding: 5px;
-}
\ No newline at end of file
+    box-sizing: border-box;
```
Comment (anewb87) [inline src/Cards.css:2]: Looking good?

File: src/Cards.css [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+}
+
+.movie-cards:hover {
+    filter: brightness(115%);
+    cursor: pointer;
+    transform: scale(1.1);
+
+}
+
+.movie-cards:focus {
```
Comment (anewb87) [inline src/Cards.css:18]: Super cool! Never heard of this before?

**Review by anewb87 — APPROVED** [review]
Nice start to the styling!

**Review by jasmyn2244 — APPROVED** [review]
We'll snag the blank lines in our final refactor. Looks great! Love the hover and focus effects

**Pull request merged**